junifer/storage/sqlite.py

```diff
@@ -235,6 +235,33 @@ def list_features(self) -> Dict[str, Dict[str, Any]]:
                 out[md5][k] = json.loads(v)
         return out
 
+    def read(
+        self,
+        feature_name: Optional[str] = None,
+        feature_md5: Optional[str] = None,
+    ) -> Dict[
+        str, Union[str, List[Union[int, str, Dict[str, str]]], np.ndarray]
+    ]:
+        """Read stored feature.
+
+        Parameters
+        ----------
+        feature_name : str, optional
+            Name of the feature to read (default None).
+        feature_md5 : str, optional
+            MD5 hash of the feature to read (default None).
+
+        Returns
+        -------
+        dict
+            The stored feature as a dictionary.
+
+        """
+        raise_error(
+            msg="read() is not implemented for SQLiteFeatureStorage.",
```

Comment:
Can you rewrite this message so that users are hinted to contact the junifer devs if they need this feature? Something like that it is with the following: "This feature is not currently planned to be implemented in the near future. If you need such functionality, contact junifer developers (LINK TO HOW TO CONTACT)"

Reply:
Maybe ask them to open an issue?

Reply:
we have a link on the doc that can be used for that.

Reply:
Are you referring to the Matrix channel link?

Reply:
https://juaml.github.io/junifer/main/help.html